### PR TITLE
Remove redundant ItemMeta initialization

### PR DESCRIPTION
**Pull request by Paul1365972**
getItemMeta already calls those methods so its redundant

### CODE REVIEW

File: src/main/java/com/mineinabyss/geary/Geary.java [hunk 1]
```diff
@@ -93,9 +93,6 @@ public void onDisable() {
   @Override
   public void attachToItemStack(Set<Component> components, ItemStack itemStack) {
     ItemMeta itemMeta = itemStack.getItemMeta();
-    if (!itemStack.hasItemMeta()) {
```
Comment (Derongan) [inline src/main/java/com/mineinabyss/geary/Geary.java:96]: getItemMeta can return null.

File: src/main/java/com/mineinabyss/geary/ecs/system/systems/ProjectileLaunchingSubSystem.java [hunk 2]
```diff
@@ -31,7 +31,7 @@ public GearyEntity launchProjectile(double speed, int projModel, Player player)
         .getHandle();
 
     ItemStack itemStack = new ItemStack(Material.SNOWBALL);
-    ItemMeta itemMeta = Bukkit.getItemFactory().getItemMeta(Material.SNOWBALL);
+    ItemMeta itemMeta = itemStack.getItemMeta();
```
Comment (Derongan) [inline src/main/java/com/mineinabyss/geary/ecs/system/systems/ProjectileLaunchingSubSystem.java:34]: Again, its a null check. We don't have a guarantee that itemMeta will exist for the itemstack, so why not be sure :P

File: src/main/java/com/mineinabyss/geary/Geary.java (hunk 1 above)
Comment (Paul1365972) [reply src/main/java/com/mineinabyss/geary/Geary.java:96]: Checked and it cant return `null` unless the material is Air and in that case `Bukkit.getItemFactory().getItemMeta` will also return null :p

From `ItemStack.java`
```java
public ItemMeta getItemMeta() {
    return this.meta == null ? Bukkit.getItemFactory().getItemMeta(this.type) : this.meta.clone();
}
```

File: src/main/java/com/mineinabyss/geary/ecs/system/systems/ProjectileLaunchingSubSystem.java (hunk 2 above)
Comment (Paul1365972) [reply src/main/java/com/mineinabyss/geary/ecs/system/systems/ProjectileLaunchingSubSystem.java:34]: See above

**Pull request closed without merging**